requirements.txt

```diff
@@ -1 +1 @@
--e .
```

Comment:
This looks like an unrelated change

Reply:
We originally changed this because we put PyYAML there but we moved that to setup.cfg. We have replaced this file with the original requirements.txt, so there is no longer any change here